.github/ci/docker/ubuntu/install-deps.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -xe
+
+source ${SPACK_ROOT}/share/spack/setup-env.sh
+spack env activate ${SENSEI_ENV}
+{
+    ls /sensei/buildcache/build_cache && \
+    spack mirror add sensei /sensei/buildcache && \
+    spack buildcache update-index sensei
+} || {
+    echo "Error: Failed to install buildcache."
+    echo "  Skipping buildcache installation..."
+}
+
+# install
+N_THREADS=$(grep -c '^processor' /proc/cpuinfo)
+spack install -v --use-cache --no-check-signature -j ${N_THREADS}
```

Comment:
I think for this we also want to add `--only dependencies`

Reply:
Ah, actually I think we want to remove the next line, considering this should also install an up to date lmod. Is the splitting between dependencies and the packages important for this script?